runners/console/index.ts

```diff
@@ -253,17 +253,22 @@ export class Console extends Runner {
         this.executeCommandSync("git clone " + command.parameters[1], directorypath, result);
 
         return result;
-
     }
 
     runNpmInstall(step: Step, command: Command): RunResult {
         let result = new RunResult();
         result.returnCode = 0;
 
         let projectPath = path.join(this.getVariable(this.workspaceDirectory), command.parameters[0]);
+        let npmCommand = "npm install";
+        if(command.parameters.length > 1){
+            if (command.parameters[1].global) npmCommand += " -g ";
+            if (command.parameters[1].args) npmCommand += command.parameters[1].args.join(" ") + " ";
+            if (command.parameters[1].name) npmCommand += command.parameters[1].name; 
```

Comment:
This would not work if only the name is given, as there is not space before the name